base/strings/util.jl

```diff
@@ -66,9 +66,11 @@ startswith(a::Vector{UInt8}, b::Vector{UInt8}) =
 # TODO: fast endswith
 
 """
-    chop(s::AbstractString)
+    chop(s::AbstractString, head::Integer=0, tail::Integer=1)
 
-Remove the last character from `s`.
+Remove first `head` and last `tail` characters from `s`.
```

Comment:
I'd have said "the first" and "the last". It would also make sense to continue saying what the default does for clarity, since that's a standard function.

Reply:
fixed